webservices/resources/elections.py

```diff
@@ -286,6 +286,7 @@ def join_candidate_totals(query, kwargs, totals_model):
         sa.and_(
             CandidateCommitteeLink.committee_id == totals_model.committee_id,
             CandidateCommitteeLink.fec_election_year == totals_model.cycle,
+            CandidateCommitteeLink.committee_designation.in_(['P', 'A']),
```

Comment:
Thanks for making this change! Like @vrajmohan mentioned in the earlier PR, it would be good to add a comment as to why this is needed, as well as add automated tests. The test should add other committee designation data and assert that they are not picked up in the totals.

Reply:
Thanks for review and good suggestion.
I will add some comment for the reason to add this condition.
for automated tests, I don't have idea how to do that.
here, committee_designation is hard code in['P','A'], committee_designation is not a filter, not in schema for output. no idea how to add other committee designation create the instance of object?

Reply:
I can pair with you to help with adding a test.